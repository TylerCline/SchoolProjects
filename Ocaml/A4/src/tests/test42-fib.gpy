def fib(x : int) : int {
    if x == 0 then 0
    else if x == 1 then 1 
    	 else fib(x-1) + fib(x-2)
}

fib(11)