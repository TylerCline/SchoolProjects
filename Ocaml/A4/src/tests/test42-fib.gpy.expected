def fib(x : int) :
 int {
 ((if (((x : int) == (0 : int)) : bool) then (0 : int) else
 ((if (((x : int) == (1 : int)) : bool) then (1 : int) else
 (((fib((((x : int) - (1 : int)) : int)) : int) + (fib((((x : int) - (2 : int)) : int)) : int)) : int)) : int)) : int)
}
(fib((11 : int)) : int)

